tests/testthat/test-CRANSKIP-helpers.R

```diff
@@ -10,13 +10,10 @@ test_that('tictoc', {
 })
 
 test_that('log with separator', {
-    expect_output(
-        cat(system("$R_HOME/bin/Rscript -e 'logger::log_with_separator(42)' 2>&1", intern = TRUE)),
-        '===')
-    expect_output(
-        cat(system("$R_HOME/bin/Rscript -e 'logger::log_with_separator(42)' 2>&1", intern = TRUE)),
-        '42')
-    expect_output(
-        cat(system("$R_HOME/bin/Rscript -e 'logger::log_with_separator(42, separator = \"|\")' 2>&1", intern = TRUE)),
-        '|||||')
+    local_test_logger(layout = layout_glue_generator("{level} {msg}"))
+
+    expect_snapshot({
```

Comment:
I think snapshots do a better job of capturing the intent here (I'm sure they didn't exist when this code was written), and since it doesn't seem to need to be run in a separate process.